sms/README.md

```diff
@@ -0,0 +1,18 @@
+# Twilio SMS integration
+
+Sign up for an account - https://www.twilio.com
+
+Requires PHP Helper Library - download here - http://github.com/twilio/twilio-php
+
+#### pantheon.yml
+
+Add this snippet to enable 
+
+```
+# Commits: Notify team of new commit to master (dev)
```

Comment:
For quicksilver-cli, snippets should be labeled `yml` and include `workflows:`.  See README for other examples.